### PR TITLE
Synchronize PC and NX Fork

### PR DESCRIPTION
**Pull request by SheridanR**

### CODE REVIEW

File: src/input.cpp [hunk 1]
```diff
@@ -40,6 +40,13 @@ void Input::defaultBindings() {
 
 	// these bindings should probably not be accessible to the player to change.
 	for (int c = 0; c < MAXPLAYERS; ++c) {
+		// NOTE disabled on public release!!!
+#ifndef PUBLIC_RELEASE_CANDIDATE
+		inputs[c].gamepad_system_bindings.insert(std::make_pair("ConsoleCommand1", (std::string("Pad") + std::to_string(c) + std::string("ButtonLeftBumper")).c_str()));
```
Comment (WALLOFJUSTICE) [inline src/input.cpp:45]: To add build envs

File: src/ui/Text.cpp [hunk 2]
```diff
@@ -95,20 +95,31 @@ void Text::render() {
...
 	Font* font = Font::get(fontName.c_str());
 	if (!font) {
 		assert(0 && "Text tried to render, but font failed to load");
 		return;
 	}
 	TTF_Font* ttf = font->getTTF();
 
+#ifdef NINTENDO
+	// fixes weird crash in SDL_ttf when string length < 2
+	std::string spaces;
```
Comment (WALLOFJUSTICE) [inline src/ui/Text.cpp:107]: Does this fix tutorial signs?

File: src/ui/MainMenu.cpp [hunk 3]
```diff
@@ -64,13 +72,13 @@ namespace MainMenu {
 		{"Spell List", "B", hiddenBinding, emptyBinding},
 		{"Skill Sheet", "K", hiddenBinding, emptyBinding},
 		{"Autosort Inventory", "R", "ButtonLeftStick", emptyBinding},
-		{"Command NPC", "Q", "DpadY-", emptyBinding},
-		{"Show NPC Commands", "C", "DpadX+", emptyBinding},
-		{"Cycle NPCs", "E", "DpadX-", emptyBinding},
+		{"Command NPC", "Q", "DpadX+", emptyBinding},
```
Comment (WALLOFJUSTICE) [inline src/ui/MainMenu.cpp:75]: Maybe better to have <- (Command NPC) ^ (Open wheel) -> (Cycle) cause it's pretty and in order to follower menu glyph layout

File: src/ui/Text.cpp (hunk 2 above)
Comment (SheridanR) [reply src/ui/Text.cpp:107]: 90% yes

**Pull request merged**